doc/manual.md.in

```diff
@@ -12,3 +12,8 @@ stdenv.md
 builders.md
 contributing.md
 ```
+
+```{=include=} appendix
```

Comment:
@infinisil it ended up as an appendix as that's what the RFC [mandates](https://github.com/7c6f434c/rfcs/blob/platform-support-tiers/rfcs/0046-platform-support-tiers.md#concrete-actions).